app/.eslintignore

```diff
@@ -0,0 +1 @@
+**/*.generated.ts
```

Comment:
no need to have node_modules/* here ?

Reply:
Per documentation, it is not neeeded
https://eslint.org/docs/1.0.0/user-guide/configuring
`In addition to any patterns in a .eslintignore file, ESLint always ignores files in node_modules/**.`